common/lib/xmodule/xmodule/modulestore/tests/test_cross_modulestore_import_export.py

```diff
@@ -345,6 +345,7 @@ def asset_collection(self):
     'manual-testing-complete',
     'split_test_module',
     'split_test_module_draft',
+    'split_test_module_static_tabs',
```

Comment:
@waheedahmed , we need to change the name of this test course--we're not testing split test modules, we're testing split mongo courses